build/script/steamcmd

```diff
@@ -1,7 +1,3 @@
 #!/usr/bin/env bash
 
-if [ `whoami` != "$STEAM_USER" ]; then
-	exec gosu $STEAM_USER $0 "$@"
-fi
-
```

Comment:
Do you think we still need it even when `docker run` and `docker exec` now run with the `steam` user by default? In what case will it help to prevent human error? Thanks for elaborating! :)